components/tyk-mdcb/values.yaml

```diff
@@ -300,5 +300,11 @@ mdcb:
     # the name of the secret
     secretName: "mdcb-tls-secret"
 
-     # the name of the volume
-    volumeName: "mdcb-tls-secret-volume"   
+    # the name of the volume
+    volumeName: "mdcb-tls-secret-volume"
+
+    # this is used to enable mtls
+    mtls:
```

Comment:
can we use this field https://github.com/TykTechnologies/tyk-charts/blob/d0ff5f3615b419772fb6062a073d0343fc13e367/components/tyk-mdcb/values.yaml#L100-L104 ?

Reply:
sure we can, only that we will end up with `.Values.mdcbSynchronizer.privateCertificateEncodingSecret`.  But the name isn't mtls specific

Reply:
we do not need to move the field into `global.mdcbSynchronizer` field. We can use `global.mtls` field as shown here https://github.com/TykTechnologies/tyk-charts/blob/d0ff5f3615b419772fb6062a073d0343fc13e367/components/tyk-mdcb/values.yaml#L100-L104

Reply:
okay